### ISSUE TITLE
Request to clarify description 

### ISSUE THREAD

**Issue by urmomsangus** [vfsfitvnm/ViMusic#146]
### Feature description

The current description says "A Jetpack Compose Android application for streaming music from YouTube Music."

I think the page would benefit from a clearer explanation that audio from YouTube videos is also playable, not just music from the Youtube Music app.

Thanks.

### Why do you want this feature?

Clearer description for how powerful the app is.

**Comment by vfsfitvnm** [comment]
It's true youtube music also has videos, but its catalogue is not equivalent to the youtube's one. If it doesn't detect music inside a video, then you can't play it. Basically, you can only listen to music. Moreover, youtube music songs are literally youtube videos.

I think "streaming music" clarifies you can't play any youtube video.

**Comment by urmomsangus** [comment]
> It's true youtube music also has videos, but its catalogue is not equivalent to the youtube's one. If it doesn't detect music inside a video, then you can't play it. Basically, you can only listen to music. Moreover, youtube music songs are literally youtube videos.
> 
> I think "streaming music" clarifies you can't play any youtube video.

Yeah, however you want to word it. 

I saw a link on Reddit with alternative music players, and ViMusic doesn't list it can play music from YouTube by itself:

https://www.reddit.com/r/AfterVanced/comments/tiqzon/youtube_music_vanced_alternatives/

I found one song from a movie soundtrack that was only available on YouTube but not YouTube Music that plays in your app successfully. But you're right the overall catalog isn't as large.

I can close this issue if you're happy with the GitHub wording. I can make a separate comment on Reddit to clarify their feature chart.

ViMusic is honestly the best, though. It's truly appreciated.
